core/conn/spj_init/src/main/resources/init_spj.sql

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+# @@@ START COPYRIGHT @@@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# @@@ END COPYRIGHT @@@
+
+SERVER_JAR=${MY_SQROOT}/export/lib/spj_init.jar
+CI_SERVER_LOG=${MY_SQROOT}/ciserver/cislog
+SQLCI=${MY_SQROOT}/export/bin32/sqlci
+CP=/bin/cp
+MANAGEABILITY_CATALOG=TRAFODION
+CIS_SCHEMA=DEFAULT_SPJ
+
+
+function dropAndCreateSchema {
+    echo "Creating Schema for CI Server Component"
+    sqlci << sqlciEOF
+
+      cqd CAT_IGNORE_ALREADY_EXISTS_ERROR 'on';
+      cqd CAT_IGNORE_DOES_NOT_EXIST_ERROR 'on';
+
+      DROP SCHEMA $MANAGEABILITY_CATALOG.$CIS_SCHEMA CASCADE;
+      CREATE SCHEMA $MANAGEABILITY_CATALOG.$CIS_SCHEMA;
+
+      cqd CAT_IGNORE_ALREADY_EXISTS_ERROR 'off';
+      cqd CAT_IGNORE_DOES_NOT_EXIST_ERROR 'off';
+
+      exit;
+sqlciEOF
+}
+
+function createProcedures {
+
+    echo "Creating Procedures in  schema '$MANAGEABILITY_CATALOG.$CIS_SCHEMA' "
+    sqlci << procEOF
+
+      cqd CAT_IGNORE_ALREADY_EXISTS_ERROR 'on';
+      cqd CAT_IGNORE_DOES_NOT_EXIST_ERROR 'on';
+
+      -- Creating Procedures 
+      set schema $MANAGEABILITY_CATALOG.$CIS_SCHEMA; 
+
+      DROP LIBRARY SPJMGMT CASCADE;
+      CREATE LIBRARY SPJMGMT FILE '${SERVER_JAR}';
+   
+      CREATE PROCEDURE HELP (
+      INOUT COMMANDNAME VARCHAR(2560) CHARACTER SET ISO88591)
+      EXTERNAL NAME 'com.traf.mgmt.JarFileMgmt.help (java.lang.String[])'
+      EXTERNAL SECURITY DEFINER
+      LIBRARY SPJMGMT
+      LANGUAGE JAVA
+      PARAMETER STYLE JAVA
+      READS SQL DATA
+      ;
+      GRANT EXECUTE ON PROCEDURE HELP TO PUBLIC;
+      
+      CREATE PROCEDURE PUT (
+      IN FILEDATA VARCHAR(102400) CHARACTER SET ISO88591,
+      IN FILENAME VARCHAR(256) CHARACTER SET ISO88591,
```

Comment:
I wonder whether we should use UTF-8 for the file names, so that we can also support file names in Chinese and other languages. Do you know whether this is supported by Linux? I would suggest VARCHAR(256 BYTES) CHARACTER SET UTF8 in that case - but then I'm not 100% sure whether SPJs support that, we might need to use UCS2?